src/main/java/tc/oc/pgm/Config.java

```diff
@@ -44,22 +44,21 @@ public static String getPath() {
   }
 
   public static class AutoRestart {
-    private final Configuration config;
 
-    public AutoRestart(Configuration config) {
-      this.config = config;
+    public static boolean enabled() {
+      return getConfiguration().getBoolean("restart.enabled", false);
     }
 
-    public boolean enabled() {
-      return this.config.getBoolean("restart.enabled", false);
+    public static Duration time() {
+      return Duration.standardSeconds(getConfiguration().getInt("restart.time", 30)); // seconds
     }
 
-    public Duration time() {
-      return Duration.standardSeconds(this.config.getInt("restart.time", 30)); // seconds
+    public static int matchLimit() {
+      return getConfiguration().getInt("restart.match-limit", 30);
     }
 
-    public int matchLimit() {
-      return this.config.getInt("restart.match-limit", 30);
+    public static Duration taskInterval() {
+      return Duration.standardSeconds(getConfiguration().getInt("restart.interval", 60)); // seconds
```

Comment:
TBH, not sure why PGM ever let this be configured, there's no reason to change it. This is okay to be hard coded.